debugger/src/main/resources/META-INF/org.jetbrains.fortran.nativeDebug.xml

```diff
@@ -0,0 +1,6 @@
+<idea-plugin>
+    <!--suppress PluginXmlValidity -->
+    <dependencies>
+        <plugin id="com.intellij.nativeDebug"/>
```

Comment:
Regarding native debug - is it possible to debug Fortran code inside IntelliJ even with the native debug plugin? 
I didn't find a way to do so.

Reply:
My understanding from reading the rust plugin is that it is possible. And that's what `nativeDebug` does in intelij Ultimate

Reply:
I will try once again, but I tried to run the sample code in IJ and failed.

Reply:
The thing is that in CLion default run configuration is "CMake application" and this run configuration is not provided by the native debug plugin.  Maybe there are some other run configurations to debug executable files with GDB, but I didn't find them in IJ+native debug. Will ask plugin authors about that

Reply:
Answer from native debug plugin authors:
"If you don't have your own run configurations native debug will not help you. The plugin contains modules that are required for debugging, but not run configurations". 
This is the difference between the Fortran plugin and the Rust one, we are re-using CLion run configurations, and they are not.
I will drop parts about native debug for now

Reply:
I am not sure about IU, but in Pycharm you can make the C interface via cmake and have a code that spans C/C++/Fortran. In that case it might be useful to have a debugger on those. But I am not sure if it can be added to all target configurations ore configured otherwise. For now dropping native debug is fine. Will have to check that there is no `:debug` included in the IU version though.